pageserver/src/http/openapi_spec.yml

```diff
@@ -520,6 +567,21 @@ components:
           type: integer
         current_logical_size_non_incremental:
           type: integer
+    WalReceiverEntry:
+      type: object
+      required:
+        - thread_id
+        - wal_producer_connstr
+      properties:
+        thread_id:
+          type: integer
+        wal_producer_connstr:
+          type: string
+        last_received_msg_lsn:
+          type: string
+          format: hex
+        last_received_msg_ts:
```

Comment:
> `_ts`

Still not sure what does this stand for, might make sense to add a doc comment (`///`) on a corresponding Rust struct field or a `description: ` field here or both.